extensions/container-image/container-image-docker/deployment/src/test/resources/openjdk-11-runtime

```diff
@@ -1,9 +1,9 @@
-FROM registry.access.redhat.com/ubi8/openjdk-11-runtime:1.16
+FROM registry.access.redhat.com/ubi8/openjdk-11-runtime:1.17
 
 ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
 
-# Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
+# Configure the JAVA_OPTS, you can add -XshowSettings:vm to also display the heap size.
```

Comment:
I think referencing `JAVA_OPTS` here, which isn't being set, is potentially misleading. How does the following sound?
 * Configure Java options. You can add -XshowSettings:vm to also display the heap size.

Reply:
Not sure if we are on the same page, yet. The semantics of `JAVA_OPTS` and `JAVA_OPTS_APPEND` are different. So if the comment refers to `JAVA_OPTS` and the actual `ENV` then uses `JAVA_OPTS_APPEND` it's a bit confusing. Could we clarify this a bit?

Perhaps we could add a brief section here what `JAVA_OPTS` does over `JAVA_OPTS_APPEND`? The former just sets what you specify in `JAVA_OPTS` ignoring all base image tuning. The latter are only in effect iff `JAVA_OPTS` isn't set and `JAVA_OPTS_APPEND` aren't set elsewhere with higher priority (e.g. deployment config). Reference doc is here:
https://jboss-container-images.github.io/openjdk/ubi8/ubi8-openjdk-11-runtime.html

So personally something like this would be clearer:

```
# Append to the Java options set by the UBI 8 OpenJDK image.  You can append '-XshowSettings:vm' to this list
# in order to also display the heap size.
ENV JAVA_OPTS_APPEND="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
```

Reply:
I tuned things a bit but be aware that these are only test resources. They are not pushed to the users at all.

If we want a comment in the generated Dockerfile, we need to apply it to `independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/base/Dockerfile-layout.include.qute` (AFAICS, there is none at the moment).

If you think it has value, I let you do it in a follow-up PR once this is in.

Thanks!

Reply:
OK, thanks for the pointer @gsmet.

Reply:
Filed: https://github.com/quarkusio/quarkus/issues/36272